python/kserve/kserve/protocol/rest/v2_datamodels.py

```diff
@@ -14,7 +14,7 @@
 from typing import Optional, List, Union, Dict
 
 import orjson
-from pydantic import BaseModel
+from pydantic import BaseModel, ConfigDict
```

Comment:
Should we surround the imports with try-catch in for backwards compatibility purposes?